portfolio/src/main/webapp/css/styles.css

```diff
@@ -10136,4 +10136,26 @@ section.resume-section .resume-section-content {
     padding-top: 5rem;
     padding-bottom: 5rem;
   }
+}
+
+.comment {
+  background-clip: padding-box;
+  background-color: #fff;
+  border-radius: 0.25rem;
+  border: 1px solid #ced4da;
+  display: block;
+  font-size: 1rem;
+  font-weight: 400;
```

Comment:
nit: The style guide suggests merging all font-.* elements together into a single font- element, for better ease of reading

Reply:
Fixed, thanks for letting me know.